AeroGearPush/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>3.0.0</string>
+	<string>3.0.7</string>
```

Comment:
The version in cocoapods was set to `3.0.9` and the project was set to `3.0.7` :grimacing:?